test/unit/test-multipolicy.cpp

```diff
@@ -44,51 +44,87 @@
 /// Source file containing tests for basic multipolicy operation
 ///
 
+#include <cstddef>
 #include "gtest/gtest.h"
 
+struct mp_tag1 {
+};
+struct mp_tag2 {
+};
+struct mp_tag3 {
+};
+
+struct mp_test_body;
+namespace RAJA
+{
+namespace impl
+{
+template <typename Iterable>
+void forall(const mp_tag1 &p, Iterable &&iter, mp_test_body const &body)
+{
+  body(p, iter.size());
+}
+template <typename Iterable>
+void forall(const mp_tag2 &p, Iterable &&iter, mp_test_body const &body)
+{
+  body(p, iter.size());
+}
+template <typename Iterable>
+void forall(const mp_tag3 &p, Iterable &&iter, mp_test_body const &body)
+{
+  body(p, iter.size());
+}
+}
+}
+
+// NOTE: this *must* be after the above to work
 #include "RAJA/RAJA.hpp"
 
+struct mp_test_body {
+  void operator()(mp_tag1 const &, std::size_t size) const
+  {
+    ASSERT_LT(size, 100);
```

Comment:
Since we are strict with type comparisons, we need to add `lu` suffix to `100` or explicitly cast to `std::size_t`. This is the only reason why travis is failing for GCC